js/models/conversations.js

```diff
@@ -245,6 +246,7 @@
       this.messageCollection.forEach(m => m.trigger('change'));
     },
     async acceptFriendRequest() {
+      // Friend request messages are always send to primary device conversation
```

Comment:
Actually friend request messages are sent to all linked devices; it's just that the receiving end shows the message as if it was sent by the sender's master device (even if it was sent from their slave device).

Reply:
I meant the "Accept / Decline" message. Updated.

Reply:
The same applies to those actually